tests/libdnf/module/ModulePackageContainerTest.cpp

```diff
@@ -4,23 +4,32 @@ CPPUNIT_TEST_SUITE_REGISTRATION(ModulePackageContainerTest);
 
 #include "libdnf/log.hpp"
 #include "libdnf/dnf-sack-private.hpp"
+#include "libdnf/hy-iutil-private.hpp"
 
 #include <algorithm>
 
+#define UNITTEST_DIR "/tmp/libdnf22XXXXXX"
+
 void ModulePackageContainerTest::setUp()
 {
     g_autoptr(GError) error = nullptr;
+    tmpdir = g_strdup(UNITTEST_DIR);
+    mkdtemp(tmpdir);
```

Comment:
There is no check of return value. When `mkdtemp` fail it will be very difficult to discover the problem.